### ISSUE TITLE
Structure page - A11y edits and function missing

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#958]
[Structure page
](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/structure.html)

**A11y:**
Link text used for multiple different destinations
The same link text is used for links going to different destinations. Users might not know the difference if they are not somehow explained.

- [x] Use the same exact description for the text links and make sure they have the same URL destination

![image (1)](https://user-images.githubusercontent.com/98193284/207426488-28948637-295a-49ce-9e1d-4b07cb18fa0a.png)
----------------------

**Function:**
- [ ] Add “Copy Code” function under source code for site function consistency

![Screen Shot 2022-12-12 at 1 11 54 PM](https://user-images.githubusercontent.com/98193284/207425596-f81c879e-ba1e-4473-8470-45581f3c86ad.png)

**Comment by kkoryaka** [comment]
@yesenia-gtz @hamiltonryan @extinah  in this case I can't add copy code because this code is display color coded code that is designed to show different pieces of code just for understanding. If button is used all kind of extra color color tags will be included that we don't want.

**Comment by yesenia-gtz** [comment]
Sounds good, we can close this one out. Thank you for the clarification  @kkoryaka

**Comment by yesenia-gtz** [comment]
Oops, there is still an a11y item needed, reopened :)

@kkoryaka

**Comment by hamiltonryan** [comment]
@extinah for that first one, I think the side nav link should match the header of the page for consistency anyway. So does "Structure" is the side nave and "Structural elements" in the page nav sound good?

**Comment by extinah** [comment]
@hamiltonryan 100% Thanks :)

**Comment by hamiltonryan** [comment]
Updated in Figma:

[Figma Link](https://www.figma.com/file/nVVip6nhCxHQnzPUnVf1bb/V6--State-Web-Template-official?node-id=2%3A3950&t=TDRoZdA8Uv3vEqDY-4)

**Comment by kkoryaka** [comment]
@sharon-rabang please review. When you are done please assign it to @pglomboy

**Comment by pglomboy** [comment]
a11y issues resolved. good to go. @kkoryaka

**Comment by sharon-rabang** [comment]
@kkoryaka Design review done, thanks!
